src/staticanalysis/bot/static_analysis_bot/config.py

```diff
@@ -65,8 +65,10 @@ def setup(self,
               cov_config=None):
         self.app_channel = app_channel
         self.download({
-            'cpp_extensions': frozenset(['.c', '.h', '.cpp', '.cc', '.cxx', '.hh', '.hpp', '.hxx', '.m', '.mm']),
+            'cpp_extensions': frozenset(['.c', '.h', '.cpp', '.cc', '.cxx', '.m', '.mm']),
+            'cpp_header_extensions': frozenset(['.h', '.hh', '.hpp', '.hxx']),
             'java_extensions': frozenset(['.java']),
+            'idl_extenssions': frozenset(['.idl']),
         })
```

Comment:
I checked the config file on MC, and those keys are not present, so we're fine here (they would be replaced otherwise)